aviatrix_controller_initialize/aviatrix_controller_init.py

```diff
@@ -284,17 +346,27 @@ def wait_until_controller_api_server_is_ready(
 
             # if the response code is 200 and the server is ready
             if is_apache_returned_200 and is_api_service_ready:
-                print(indent + keyword_for_log + "Server is ready")
+                print(indent + keyword_for_log + "Server is ready", flush=True)
                 return True
         except Exception as e:
```

Comment:
Whenever Exception is excepted we need to use logger.exception() or print(trackback.format_exec())